### ISSUE TITLE
Invalid setup guide v6.1

### ISSUE THREAD

**Issue by WinterSilence** [symfony/symfony-docs#16551]
Command `symfony new symfony.test --version=6.1 --webapp` taken from [online docs](https://symfony.com/doc/6.1/setup.html) is failed: can't find package for 6.1
Also, `symfony` can't detect Composer by `$_ENV['COMPOSER_HOME']`.

**Comment by htmlshaman1** [comment]
status: not working
`symfony new symfony.test --version=6.1 --webapp`

**Comment by WinterSilence** [comment]
@htmlshaman1 what's you mean?

**Comment by htmlshaman1** [comment]
I tried the same code you ran and it didn't work.  I'm just confirming it isn't working.

```
 [InvalidArgumentException]                                   
  Could not find package symfony/skeleton with version 6.1.*.
```

**Comment by xabbuh** [comment]
Does it work when you use `symfony new symfony.test --version=next --webapp` instead?

**Comment by WinterSilence** [comment]
@xabbuh if it's work for you, then it's work for anybody

**Comment by htmlshaman1** [comment]
This works:
`symfony new symfony.text --version=next --webapp`

**Comment by javiereguiluz** [comment]
We're trying to fix this in #16592. Thanks!

**Comment by javiereguiluz** [comment]
Closing as fixed by #16592.

**Comment by WinterSilence** [comment]
@javiereguiluz invalid solution - 6.1 is "next" only now and what's about 6.0 manual(same problem)?

**Comment by xabbuh** [comment]
@WinterSilence `symfony new symfony.test --version=6.0 --webapp` works for me

**Comment by javiereguiluz** [comment]
@WinterSilence yes, when .1 is released, we'll update the docs to use `6.1` here and `next` in the new 6.2 branch. We've always done the same in the past. I don't know why `next` was removed here (but I guess it was probably an error of mine when merging things from lower to upper branches).
